transfers/transfer.py

```diff
@@ -224,29 +227,40 @@ def run_scripts(directory, *args):
             continue
         script_name, script_ext = os.path.splitext(script)
         if script_extensions and script_ext not in script_extensions:
-            LOGGER.info("'%s' for '%s' not in configured list of script file extensions, skipping", script_ext, script_path)
+            LOGGER.info(("'%s' for '%s' not in configured list of script file "
+                         "extensions, skipping", script_ext, script_path))
             continue
         LOGGER.info('Running %s "%s"', script_path, '" "'.join(args))
-        p = subprocess.Popen([script_path] + script_args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        p = subprocess.Popen(
+            [script_path] + script_args, stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE)
         stdout, stderr = p.communicate()
         LOGGER.info('Return code: %s', p.returncode)
         LOGGER.info('stdout: %s', stdout)
         if stderr:
             LOGGER.warning('stderr: %s', stderr)
 
 
-def get_next_transfer(ss_url, ss_user, ss_api_key, ts_location_uuid, path_prefix, depth, completed, see_files):
+def get_next_transfer(ss_url, ss_user, ss_api_key, ts_location_uuid,
+                      path_prefix, depth, completed, see_files):
     """
-    Helper to find the first directory that doesn't have an associated transfer.
+    Helper to find the first directory that doesn't have an associated
+    transfer.
 
     :param ss_url: URL of the Storage Sevice to query
     :param ss_user: User on the Storage Service for authentication
-    :param ss_api_key: API key for user on the Storage Service for authentication
+    :param ss_api_key: API key for user on the Storage Service for
+                       authentication
     :param ts_location_uuid: UUID of the transfer source Location
     :param path_prefix: Relative path inside the Location to work with.
-    :param depth: Depth relative to path_prefix to create a transfer from. Should be 1 or greater.
-    :param set completed: Set of the paths of completed transfers. Ideally, relative to the same transfer source location, including the same path_prefix, and at the same depth.
-    :param bool see_files: Return files as well as folders to become transfers.
+    :param depth: Depth relative to path_prefix to create a transfer from.
+                  Should be 1 or greater.
+    :param set completed: Set of the paths of completed transfers. Ideally,
+                          relative to the same transfer source location,
+                          including the same path_prefix, and at the same
+                          depth.
+    :param bool see_files: Return files as well as folders to become
+                           transfers.
```

Comment:
While I personally appreciate respecting the 80 character line limit, most of AM's codebase does not respect it, so it's probably not practical to fix all of the >80 chars lines in here. If you are going to do embark on massive refactors, it would make it a lot easier to code review if you separate content-ful commits from purely stylistic ones. Then the reviewer can go through the stylistic commit(s) and just confirm that nothing has been changed in terms of semantics/logic.

Reply:
I've given it another shot to make this section look a bit prettier and still respect the line limit (this is a tough script to make it adhere to that!). I hadn't anticipated that style/contents should be separated but will consider this in future.